ptp-tools/scripts/build-push-deploy.sh

```
@@ -0,0 +1,366 @@
+  make -C "$TOOLS_DIR" deploy-all
+
+  if [[ -n "$PTPOP_SPEC" ]]; then
+    echo ""
+    echo "=== Restarting operator to pick up new image ==="
+    kubectl rollout restart deployment/ptp-operator -n openshift-ptp
+    kubectl rollout status deployment/ptp-operator -n openshift-ptp --timeout=120s
+  fi
+
+  echo ""
+  echo "=== Waiting for operator to reconcile daemonset ==="
+  for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)